### ISSUE TITLE
Dialer Server Crash

### ISSUE THREAD

**Issue by whatthedrunk** [McJty/RFToolsDimensions#77]
Trying to narrow down why this is happening but it seems when anyone on the server access their dialer the server locks up then crashes. Going though the process of removing things like foamfix but no luck so far.

http://pastebin.com/H5xzJhqg

**Comment by McJty** [comment]
The crash occurs in teamroots.embers.network.message. I don't know what that is but that's not part of rftools. Can it be some other mod is making things break here?

**Comment by whatthedrunk** [comment]
Yeah that's what I was reading too but he wasn't sure it was actually related to embers/roots or now. Took us a while to figure out its only when someone looks at their dialer though. I can cause it to crash every time by looking at the GUI.

**Comment by whatthedrunk** [comment]
It may just be an embers problem but if you want to look at the logs here is one from the crash. https://drive.google.com/drive/folders/0ByQjJOciCO7leUdEcFQwYVFfQlk?usp=sharing
